conf/local.conf.sample

```diff
@@ -280,3 +280,14 @@ BB_DISKMON_DIRS = "\
 # track the version of this file when it was generated. This can safely be ignored if
 # this doesn't mean anything to you.
 CONF_VERSION = "1"
+
+# WIC image type support
+IMAGE_FSTYPES ?= "tar.bz2 wic.bz2"
+
+# dosfstools is needed to create the boot partition, mtools to copy to it
+IMAGE_DEPENDS_wic_append = " dosfstools-native mtools-native"
+
+# Uncomment one of these for specific SD card sizes (default=4g)
+WKS_FILE ?= "sdimage-bootpart-4g.wks"
+# WKS_FILE ?= "sdimage-bootpart-2g.wks"
+# WKS_FILE ?= "sdimage-bootpart-8g.wks"
```

Comment:
I don't think this is appropriate. Using generic wks files is good, but not every MACHINE is guaranteed to work with them. I think this block still belongs in the machine .conf files, or perhaps a .inc which the machines include. For example, I'm pretty sure this is just plain wrong for the qemu machines, which we have to support.